sourcegraph/about
# Update onboarding use cases page

Add feedback from the [doc](https://docs.google.com/document/d/1HR3O5IL1dNtoME0-b3jfSho7Q-yQrOqHJd5r-UjTv88)

## To Test
- Check out the onboarding use cases page and ensure that: logos are no longer company links and the copy changes in the carousel match the requirements in the doc.
